packages/core/test/chainbridge/helpers.js

```diff
@@ -0,0 +1,126 @@
+/**
```

Comment:
Are these tests just testing the chainbridge code and/or replicating chainbridge tests? Do we need to test it if they already are? Or is this testing specific use cases that we have in mind?

Reply:
This file is copied from the chainbridge repo. The `voting.js` file is the only one that I created and it demonstrates how chainbridge would work with our Voting contract

Reply:
Are these tests for that file?

Reply:
Oh never mind, I see. Voting.js is a test.